ssh/id_busybee

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Yes, `make setup` copies these files to host:~/.ssh so that it is possible to ssh into the st2-upstart container.